terraform/modules/supernode/variables.tf

```diff
@@ -8,7 +8,7 @@ variable "region" {
 
 variable "amis" {
   default = {
-    "us-east-1" = "ami-06a556b364d73800b"
+    "us-east-1" = "ami-0ac019f4fcb7cb7e6"
```

Comment:
Looks like this is custom/private image? Are there a scripts and/or documented steps so we can create it ourselves?

Reply:
> 
> 
> Looks like this is custom/private image? Are there a scripts and/or documented steps so we can create it ourselves?

packer's json file has been provided here
Steps to create image:    
    cd packer
    $ packer build -var 'aws_access_key=XXXXXXXXXXXXXXXXXXXXXXX' -var 'aws_secret_key=YYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY' ami.json
adding subsection for packer to UG

Reply:
thanks, it's clear